cmd/viewer/main.go

```diff
@@ -89,6 +91,10 @@ func run() error {
 	app.Get("/networks/:network/blocks", viewerHandler.ListBlocks)
 	app.Get("/networks/:network/blocks/:blockHash", viewerHandler.GetBlock)
 
+	app.Get("/transactions", viewerHandler.ListTransactions)
+	app.Get("/transactions/:txHash", viewerHandler.GetTransactionByTxHash)
+	app.Get("/transactions/:blockNum", viewerHandler.GetTransactionsByBlockNum)
+
```

Comment:
The route here should conflict and the parameters can be put into query based on a `txHash` or `blockNumber` search.
such as `/transaction?txHash=xx&blockNumber=xxx`.